transport/dlr-malspp/transport/src/main/java/org/ccsds/moims/mo/testbed/util/sppimpl/util/SPPReader.java

```diff
@@ -147,25 +189,32 @@ public synchronized SpacePacket receive() throws IOException
 //        outPacket.setBody(trimmedBody);
     // Read CRC
     if (crcEnabled) {
+
       int CRC = SPPHelper.computeCRC(inHeaderBuffer, data, outPacket.getOffset(), dataLength);
```

Comment:
CRC should not be calculated if it's not whitelisted

Reply:
I'll restructure it!

Reply:
Done.